### ISSUE TITLE
Feature Request: In-line help for parameters

### ISSUE THREAD

**Issue by pcgeek86** [PowerShell/PowerShellEditorServices#202]
I would recommend that we include in-line help for parameters, to minimize the need for someone to separately call `Get-Help`. We've already got Intellisense for the parameter name & type, but if we could inject the help statement for the parameter, that would be golden.

![2016-03-29 13_12_50- untitled-1 - visual studio code - insiders](https://cloud.githubusercontent.com/assets/466713/14120450/20451a20-f5b0-11e5-9f3a-c0199201e7e1.png)

Cheers,
**Trevor Sullivan**
https://trevorsullivan.net
https://twitter.com/pcgeek86

**Comment by daviwil** [comment]
Absolutely, I had wanted to add this early on in development but got busy and never filed an issue for it. We should definitely add this in the near future.

**Comment by andyleejordan** [comment]
Duplicate of #2241
